runbotics/runbotics-orchestrator-ui/src/src-app/components/Tile/ProcessTile/ProcessTileTagList/ProcessTileTagList.tsx

```diff
@@ -69,9 +68,17 @@ const ProcessTileTagList: FC<ProcessTileTagListProps> = ({
 
     const tagNameFitSearch = (tagName: string) => Boolean(tagName.match(RegExp(searchValue, 'ig')));
 
+    useEffect(() => {
+        setExpandedHeight(refProcessTileContent.current.offsetHeight + 32);
```

Comment:
why + 32? If you have to use it make it named const

Reply:
It is a margin of process content div.